Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: all
+all:
```

Comment:
isn't a Makefile built exactly to mark dependencies? wouldn't you just put `build test pack` as a list and it would run those steps before running `all` ?

Reply:
so that line would look like: `all: build test pack`
I'm still unsure what the freaking .PHONY thing does :)